css/image-container/image-container.md

```diff
@@ -0,0 +1,69 @@
+## Image-Container
+
+Sometimes we need to use containers for displaying images in html, in order 
+to properly position and limit their dimensions. We can provide certain 
+width and height to the image, but for every variation of a container, 
+we need to specify the dimensions which may result excessive 
+repetition of css code.
+
+
+### Fixed Size Container
+
+If we have a fixed sized container we can use full width and height.
+
+```css
+.image-container-fixed img {
+    width: 100%;
+    height: 100%;
+}
+```
+
+With this way, the image fully matches the dimensions of the container, 
+however the aspect ratio of the image may be lost. We can alter the _object-fit_ 
+property to protect the aspect ratio to overcome that issue. 
+
+#### Margin/Padding Issue
+
+A problem occurs with this approach is, if we want some margin or padding for 

```

Comment:
"A problem that occurs with this approach is, ..." veya "A problem occurs with this approach, ..." daha doğru olur gibi geldi bana